### PR TITLE
Merge CSS changes

### PR DESCRIPTION
**Pull request by mogeritz**
Review and test changes to CSS to be able to merge to main

### CODE REVIEW

File: styles/page.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    margin-top: 5rem;
+    background-color: #f0ece7;
+    color: black;
+    width: 100%;
+    font-size: 1.8rem;
+}
+/* End of General Elements Styling */
+
+/* Page Identifier Name */
+main {
```
Comment (vchou105) [inline styles/page.css:23]: should main be moved to general elements styling? I wrote it here at that time but can other people share their opinons

File: index.html [hunk 2]
```diff
@@ -4,49 +4,62 @@
...
         <section>
             <table>
                 <!-- DO NOT TOUCH -->
                 <thead>
                     <tr>
-                    <th>Name</th>
-                    <th>Estimated Time</th>
-                    <th>Status</th>
-                    <th>Action</th>
+                        <th id="theadName">Name</th>
```
[the remark this answers is not in the document]
Comment (melvyn9) [reply index.html:37]: Above ^

File: styles/page.css (hunk 1 above)
Comment (melvyn9) [reply styles/page.css:23]: Yes it should be moved to general styling.

**Review by melvyn9 — APPROVED** [review]
There are some subtle problems in HTML and CSS. But that can be resolved later.

**Review by vchou105 — APPROVED** [review]

**Pull request merged**